assets/src/styles/style.scss

```diff
@@ -6,3 +6,6 @@
 @import "styleguide/src/base/fonts";
 
 @import "blocks";
+
+// Third-party vendor overrides / customizations
+@import "vendors/photoswipe/planet4-photoswipe";
```

Comment:
This seems like a good candidate to load at the body end, since we already have way to much blocking CSS, and I guess for now we don't have a way to open the lightbox that quickly :smile:

Reply:
Indeed, I was hesitant about adding more CSS (and JS) to the bundle, I considered loading them using the hooks from the Media PR (useStylesheet...), but as we need it for image blocks and that is pretty much on all pages, I left them in the bundle. 

I'll move the CSS away and enqueue separately. As for the JS stuff, in the future we should consider unbundling the blocks and loading only the ones we need on each page.